docs/SimpleList.md

```diff
@@ -0,0 +1,30 @@
+| addFilter         | Function                                                   | undefined     | false    | returns some JSX to render an additional filter                                                                                                                                                                                                                                                                      |
+| bgcAlt            | boolean                                                    | false         | false    | if true, appends `baseClass` with `--bgc-alt`                                                                                                                                                                                                                                                                        |
+| clickAct          | Function                                                   | null          | false    | Callback function that is trigger clicking on a row                                                                                                                                                                                                                                                                  |
+| filterOpts        | Array&lt;{<br>  text: string,<br>  value: string,<br>}&gt; | []            | false    | Array of object that are mapped to create filter options in the `<FilterComponent />`.<br>Value is placed in `predefined` prop of `<FilterComponent />`<br>When clicking on the `Filter Component` they are appear for selection<br>Clicking on a `Filter Option` will append the value to the search bar            |
+| tableTitle        | string                                                     | ''            | false    | String of `SimpleList` table                                                                                                                                                                                                                                                                                         |
+| sections          | boolean                                                    | false         | false    | If true `<Sections />` component will be rendered and if false `<Body />`<br> component will be rendered                                                                                                                                                                                                             |
+| sectionTarget     | string                                                     | ''            | false    | TODO: investigate the purpose of `sectionTarget`                                                                                                                                                                                                                                                                     |
```

Comment:
@robguy21 can you add your comment here

Reply:
"This is to be used alongside the `sections` prop, it is used to isolate the data for the current inner list, therefore it will be the key of `sections` where `sections` is in the shape: { string: Array<Object> } "